resolwe_bio/processes/expression/bowtie2_quant.yml

```diff
@@ -0,0 +1,137 @@
+# =======================
+# Expression from bowtie2
+# =======================
+---
+
+- slug: bowtie2-quant
+  name: Quantify species from bowtie2 output
```

Comment:
Maybe better: `Quantify shRNA species`

Reply:
I would prefer to have bowtie2 in the description, so my suggestion would be

> Quantify shRNA species using bowtie2